### ISSUE TITLE
DocumentCard: Examples should have aria-label attribute specified

### ISSUE THREAD

**Issue by khmakoto** [OfficeDev/office-ui-fabric-react#10664]
The `DocumentCard` examples in [the website](https://developer.microsoft.com/en-us/fabric#/controls/web/documentcard) don't read properly when using screen readers because the `aria-label` attribute is not being defined. The examples should define this attribute so that users of screen readers have a good experience when navigating through the website.

#### Notes
Bug 4 in #10646 - DocumentCard A11y Pass
